src/background.js

```diff
@@ -24,8 +24,8 @@ chrome.pageAction.onClicked.addListener(function(){
 });
 
 chrome.tabs.onUpdated.addListener(function(tabId, changeInfo, tab){
-  const regexUrl = /https:\/\/github\.com\/.*\/.*\/compare\/.*/;
-  if(tab.url.match(regexUrl)){
+  const regexUrl = /https:\/\/github\.com\/.*\/.*\/compare\/..*/;
+  if(regexUrl.test(tab.url) && changeInfo.url){
```

Comment:
instead of testing tab.url against the regexp, does it make more sense to just test changeInfo.url?